CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [4.0.0](https://github.com/algolia/angular-instantsearch/compare/4.0.0-alpha.4...4.0.0) (2021-08-31)
+
+
```

Comment:
this should have the actual changes (a link to the docs) and maybe the alphas removed?

Reply:
The only actual change was update to IS 4.28, but it didn't come up as it's a chore.
Alpha removed is already in the version no?

Reply:
Ah, I wasn't clear about this during the zoom call earlier. You need to put stuff between v3.x and 4.0.0 there and remove the alpha changelogs.

Reply:
yes, I think that's best

Reply:
ok done!